EXAMPLES.md

```diff
@@ -2,343 +2,389 @@
-                                 --email account@test.com \
-                                 --password accountpassword --host 127.0.0.1
+
+vaultclient generate-account-access-key --name john
 
 ```
 
-#### delete-account
+The previous example will generate a new account access key for the account
+named ```john```
 
-```sh
-$ bin/vaultclient delete-account --name TestAccount --host 127.0.0.1
+Output example :
+
+```json
 
 {
-    "message": {
-        "code": 204,
-        "message": "No content."
-    }
+    "id": "XMHR9IQ9UYN56W1OSN2S",
+    "value": "5tK4XOid7pXss66A7Jn=Yz7ybnMIB4Uf/BjavN58",
+    "createDate": "2016-07-02T21:47:57Z",
+    "lastUsedDate": "2016-07-02T21:47:57Z",
+    "status": "Active",
 }
-```
-
-### Users
 
-NB: to create a user, you need to have created an account to which the user
-will belong
+```
 
-#### create-user
+### List accounts
 
 ```sh
-$ bin/vaultclient create-user --account-name TestAccount --name TestUser \
-                              --email user@test.com --password userpassword \
-                              --host 127.0.0.1
 
-{
-    "message": {
-        "code": 201,
-        "message": "Created",
-        "body": {
-            "arn": "arn:aws:iam::117099473272:/TestUser/",
-            "id": "5EF3TQIXSXP3QFLF8783TT8ZLO37XJLK",
-            "name": "TestUser",
-            "createDate": "2016-02-22T11:24:39+01:00"
-        }
-    }
-}
+vaultclient list-accounts --max-items 30
+
 ```
 
-NB: User names can include spaces in any command if double quotes are used:
+The previous example will list at most, 30 accounts. You can also paginate by
+using ```--marker```, like :
 
 ```sh
-$ bin/vaultclient create-user --account-name "Account name with spaces" \
-                              --name "User name with spaces" \
-                              --email user@test.com \
-                              --password userpassword \
-                              --host 127.0.0.1
+
+vaultclient list-accounts --max-items 30 --marker 1
+
 ```
 
-#### delete-user
+The previous example will list at most, the 30 next accounts in the list.
 
-```sh
-$ bin/vaultclient delete-user --account-name TestAccount --name TestUser \
-                              --host 127.0.0.1
+Output example:
+
+```json
 
 {
-    "message": {
-        "code": 204,
-        "message": "No content."
-    }
+    "isTruncated": false,
+    "accounts": [
+        {
+            "arn": "arn:aws:iam::341220772100:/john/",
+            "id": "341220772100",
+            "name": "john",
+            "createDate": "2016-07-02T21:47:53Z",
+            "emailAddress": "john@acme.com",
+            "canonicalId": "UO62HEUU76W5LYMG41SO3MQZQQN21YGQ1ZSF25B47GNUCC5F1"
+        },
+        {
+            "arn": "arn:aws:iam::148910879031:/test2/",
+            "id": "148910879031",
+            "name": "test2",
+            "createDate": "2016-07-02T21:59:27Z",
+            "emailAddress": "test2@test.com",
```

Comment:
this needs to be changed to from
`test2` to `jane`
 `test2@test.com` to `jane@acme.com`.